modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/plugin/ArtifactPluginDescriptorFactory.java

```diff
@@ -142,8 +142,7 @@ protected void doDescriptorConfig(MulePluginModel artifactModel, ArtifactPluginD
   @Override
   protected ArtifactPluginDescriptor createArtifactDescriptor(File artifactLocation, String name,
                                                               Optional<Properties> deploymentProperties) {
-    // Keep compatibility with usages of the factory that expect the descriptor from previous version.
-    return new ArtifactPluginDescriptor(name, deploymentProperties);
+    return new ArtifactPluginDescriptor(name);
```

Comment:
Given that the new `ArtifactPluginDescriptor` class from `artifact` module is being used since March, we could maybe even delete the old one?

Reply:
if it is not used by the agent or tooling, yes!